coalib/misc/Caching.py

```diff
@@ -14,8 +14,15 @@ class FileCache:
 
     >>> from pyprint.NullPrinter import NullPrinter
     >>> from coalib.output.printers.LogPrinter import LogPrinter
+    >>> from coalib.output.Logging import configure_logging
     >>> import logging
     >>> import copy, time
+    >>> configure_logging()
+    20
+
+    configure_logging should always be run before creating a
```

Comment:
Maybe this could go into `LogPrinter`'s `__init__`? (Basically if logging hasn't been configured yet, do it at object creationg.)

Not sure how it'd affect other things though.